maistra/README.md

```diff
@@ -0,0 +1,95 @@
+# Maistra
+
+## Fork
+
+This repository is a fork of the [Envoy project](https://github.com/envoyproxy/envoy), and it is modified to meet certain criteria so it can be used as the base for the OpenShift Service Mesh product by Red Hat.
```

Comment:
not sure what you mean

Reply:
reading this back, me neither. I remember clicking through on the link and thinking I landed on the project page instead of the repo, but trying again that's not the case. please ignore.